src/standalone/Snow/Snow.jl

```diff
@@ -500,6 +498,71 @@ function ClimaLand.get_drivers(model::SnowModel)
     )
 end
 
+"""
+    function ClimaLand.turbulent_fluxes!(
+        dest,
+        atmos::PrescribedAtmosphere,
+        model::SnowModel,
+        Y::ClimaCore.Fields.FieldVector,
+        p::NamedTuple,
+        t,
+    )
+
+A SnowModel-specific function to compute turbulent fluxes with a
+prescribed atmosphere; returns the latent heat flux, sensible heat flux,
+vapor flux, and aerodynamic resistance.
+
+We cannot use the default version in src/shared_utilities/drivers.jl
+because the snow model must first compute the specific humidity at the
+surface based on the atmospheric surface air density and the temperature
+of the snow surface.
+"""
+function ClimaLand.turbulent_fluxes!(
+    dest,
+    atmos::PrescribedAtmosphere,
+    model::SnowModel,
+    Y::ClimaCore.Fields.FieldVector,
+    p::NamedTuple,
+    t,
+)
+    # Compute specific humidity before turbulent fluxes
+    @. p.snow.q_sfc = snow_surface_specific_humidity(
+        p.snow.T_sfc,
+        p.snow.q_l,
+        p.drivers.thermal_state,
+        model.parameters,
+    )
+
+    T_sfc = surface_temperature(model, Y, p, t)
+    ρ_sfc = surface_air_density(atmos, model, Y, p, t, T_sfc)
+    q_sfc = surface_specific_humidity(model, Y, p, T_sfc, ρ_sfc)
```

Comment:
Could we maybe move this:
``` 
@. p.snow.q_sfc = snow_surface_specific_humidity(
        p.snow.T_sfc,
        p.snow.q_l,
        p.drivers.thermal_state,
        model.parameters,
    )
```

into `surface_specific_humidity` for snow? We might not need an additional method, then.

Reply:
Ahh yes! Good point